frontend/src/app/projects/models/backup-settings/pipeline-deletion-dialog/pipeline-deletion-dialog.component.html

```diff
@@ -0,0 +1,43 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div class="dialog">
+  <h2 class="mb-2 text-xl font-medium">
+    Remove a TeamForCapella to Git backup pipeline
+  </h2>
+  <p>
+    Do you want to delete the TeamForCapella to Git backup pipeline
+    {{ data.backup.id }}?
+  </p>
```

Comment:
As a precaution, this should include more information about the pipeline instead of just the id to prevent accidentally deleting the wrong one.

Reply:
Contains more information now: 

![image](https://github.com/user-attachments/assets/9217f7a1-ea2d-4e3a-83f3-31a20d28f3ab)